app/views/content_items/manual_section.html.erb

```diff
@@ -49,21 +49,48 @@
                 </div>
               <% end %>
             <% else %>
-              <%= render "govuk_publishing_components/components/accordion", {
-                anchor_navigation: true,
-                items: @content_item.main.map do |item|
+              <%
+                items = @content_item.main.each.with_index(1).map do |item, index|
                   rendered_content = render "govuk_publishing_components/components/govspeak", {} do
                     raw(item[:content])
                   end
 
                   {
+                    data_attributes: {
+                      ga4: {
+                        event_name: "select_content",
+                        type: "accordion",
+                        text: item[:heading],
+                        index: index,
+                        index_total: @content_item.main.length,
+                      }
+                    },
                     heading: item[:heading],
                     content: {
                       html: rendered_content,
                     },
                   }
                 end
-              } %>
+              %>
+
+              <div data-module="gtm-click-tracking">
```

Comment:
`gtm-click-tracking` can be passed directly to the component as a `module` value in `data_attributes`.